build/media-suite_compile.sh

```diff
@@ -1737,6 +1750,11 @@ if [[ $ffmpeg != "no" ]]; then
                 do_removeOption --enable-libsvthevc
         fi
 
+        if enabled libsvtav1; then
+            do_patch "https://raw.githubusercontent.com/OpenVisualCloud/SVT-AV1/master/ffmpeg_plugin/0001-Add-ability-for-ffmpeg-to-run-svt-av1-with-svt-hevc.patch" am ||
```

Comment:
What's with this patch name. It makes it seem like svt-av1 depends on svt-hevc.

Reply:
There are 2 patches by them, I believe this one is needed if we want ffmpeg to build with both svt-hevc and svt-av1.

Reply:
Well, it is in fact that way in a sense. It assumes the svt-hevc patch is also applied and will fail to be applied otherwise.
You should add a check that svt-hevc is also enabled.

Reply:
It's because since neither of them are merged in ffmpeg, due to the both of the libraries being placed in the same spot in the configure file, they would cause a conflict, so they created another patch with the context including svt-hevc,

Reply:
maybe I can change it so that if svthevc is enabled and svtav1, then apply that one, else apply the regular one

Reply:
Yeah.